Agilefreaks/FreaksCatalog-Android
# Update to the latest gradle version

#### Description

Update suggested by the latest version of Android Studio

#### [Task](task_url)

#### Related PRs




## Steps to Test or Reproduce

- make sure Android Studio Artic Fox is installed
- gradle 7 requies java 11, it should come with Android Studio, you can also install it with sudo apt install openjdk-11-jdk
- run the project

## Risks
